pex/build_system/pep_517.py

```diff
@@ -183,8 +180,8 @@ def _invoke_build_hook(
             args=args,
             env=build_system.env,
             cwd=project_directory,
-            stdout=stdout if stdout is not None else sys.stderr.fileno(),
-            stderr=stderr if stderr is not None else sys.stderr.fileno(),
+            stdout=subprocess.PIPE,
```

Comment:
Working through an issue with non-reproducible local project locks when fiddling with the proper `setup.{cfg,py}` setup (see various comments in those files) I became keenly aware of all the sdist build output leaked to `stdout` / `stderr` here. No callers passed `stdout` / `stderr` and always piping does the right thing since `Job` knows to slurp the streams up and only report output when the `Job` fails, which is exactly what you want for a non-noisy CLI that is only noisy when it needs to be to aid debug of a build failure.